modules/openapi-generator/src/main/resources/kotlin-client/libraries/jvm-volley/api.mustache

```diff
@@ -0,0 +1,129 @@
+package {{apiPackage}}
+
+import android.content.Context
+import com.android.volley.DefaultRetryPolicy
+import com.android.volley.Request
+import com.android.volley.RequestQueue
+import com.android.volley.Response
+import com.android.volley.toolbox.BaseHttpStack
+import com.android.volley.toolbox.Volley
+import java.util.*;
+import kotlin.coroutines.resume
+import kotlin.coroutines.resumeWithException
+import kotlin.coroutines.suspendCoroutine
+import com.google.gson.reflect.TypeToken
+
+import {{packageName}}.request.IRequestFactory
+import {{packageName}}.request.RequestFactory
+import {{packageName}}.infrastructure.CollectionFormats.*
+
+{{#imports}}import {{import}}
+{{/imports}}
+
+{{#operations}}
+/*
+* If you wish to use a custom http stack with your client you
+* can pass that to the request queue like:
+* Volley.newRequestQueue(context.applicationContext, myCustomHttpStack)
+*/
+class {{classname}} (
+    val context: Context,
+    val requestQueue: Lazy<RequestQueue> = lazy(initializer = {
+        Volley.newRequestQueue(context.applicationContext)
+    }),
+    val requestFactory: IRequestFactory = RequestFactory(),
+    val basePath: String = "{{{basePath}}}",
+    private val postProcessors :List <(Request<*>) -> Unit> = listOf()) {
+
+    {{#operation}}
+    /**
+    * {{summary}}
+    * {{notes}}
+    {{#allParams}}   * @param {{paramName}} {{description}}
+    {{/allParams}}   * @return {{#returnType}}{{{returnType}}}{{/returnType}}{{^returnType}}void{{/returnType}}
+    */
+    {{#isDeprecated}}
+    @Deprecated("This api was deprecated")
+    {{/isDeprecated}}
+    suspend fun {{operationId}}({{^allParams}}){{/allParams}}{{#allParams}}{{>queryParams}}{{>pathParams}}{{>headerParams}}{{>bodyParams}}{{>formParams}}{{^-last}}, {{/-last}}{{#-last}}{{#operationAuthMethod}}, opAuthHeaderFactory = () -> map<String, String>{{/operationAuthMethod}}){{/-last}}{{/allParams}}: {{#returnType}}{{{returnType}}}?{{/returnType}}{{^returnType}}Unit{{/returnType}} {
+        {{#bodyParam}}
+        var body: Any? = {{paramName}}
+        {{/bodyParam}}
+        {{^bodyParam}}
+        var body: Any? = null
+        {{/bodyParam}}
+        {{#allParams}}
+        {{#required}}
+        // verify the required parameter '{{paramName}}' is set
+        // This is probably taken care of by non-null types anyway
+        requireNotNull({{paramName}})
+        {{/required}}
+        {{/allParams}}
+
+        var contentTypes : Array<String> = arrayOf({{#consumes}}"{{{mediaType}}}"{{^-last}},{{/-last}}{{/consumes}})
+        var contentType: String = if (contentTypes.size > 0) { contentTypes[0] } else { "application/json" }
```

Comment:
@alisters could you replace var with val?
Could you please update the if else to `if (contentTypes.isNotEmpty()) { contentTypes.first() } else { "application/json" }`?

Reply:
Done